src/ui/EditorNS/jsproxy.cpp

```diff
@@ -0,0 +1,116 @@
+#include "include/EditorNS/jsproxy.h"
+#include <QDebug>
+
+namespace EditorNS {
+
+QVariant JsToCppProxy::getResult()
+{
+    return m_result;
+}
+
+QVariant JsToCppProxy::getCursor()
+{
+    return m_cursor;
+}
+
```

Comment:
Think you can figure out a way to do the type conversion within the proxy still?   I would much prefer your method, but I don't want type conversion happening within the editor if we can help it.